common/persistence/src/test/java/app/coronawarn/server/common/persistence/service/TraceTimeIntervalWarningServiceTest.java

```diff
@@ -181,9 +182,9 @@ void testSaveCheckInProtectedReports() {
   void testGetCheckInProtectedReports() {
```

Comment:
Feel like this test is at the wrong place. It is a test tracetimeintervalwarningSERVICE but you are testing the repository. So I would suggest adding a test for SERVICE layer there

Reply:
It seems to me that testSaveCheckInProtectedReports -> due to the fact that it checks the increased integer, it actually testing the service. I may be wrong, let's talk about it

Reply:
okay I agree, but then let's rename the test. Something like shouldReturnCheckInWithCorrectSubmissionOrder